src/engine/sys/sdl_glimp.cpp

```diff
@@ -575,8 +572,29 @@ static rserr_t GLimp_SetMode( int mode, bool fullscreen, bool noborder )
 		logger.Warn( "GLEW version < 2.0.0 doesn't support GL core profiles" );
 	}
 
+	int colorBits = std::max( 0, r_colorbits->integer );
+	// int alphaBits = std::max( 0, r_alphabits->integer );
```

Comment:
commented-out code

Reply:
This is the code I mention in my PR description:

> Unless I did a mistake, the value of `r_ext_multisample` and `r_alphabits` appears to be unused.

Also this code was rewritten. It's possible I kept that comment just to remember that is the place of this code if we need it (maybe not, I don't know). I believe multisample has to be implemented, but I don't know what alphabit is for.